css/style.css

```diff
@@ -291,4 +291,63 @@ html {
   height: 55rem;
   border-radius: 100%;
   box-shadow: 0 3rem 5rem rgb(0, 0, 0, 0.3);
+}
+
+/******************************/
+/* INGREDIENTS SECTION */
+/******************************/
+
+.ingredients-section {
+  background: url('../gallery/banners/ingredients-banner.webp');
+  background: url('../gallery/banners/ingredients-banner.jpg');
+  background-size: cover;
+  width: 100%;
+  padding: 10rem 0;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.dark-square {
+  width: 57rem;
+  background-color: rgba(0, 0, 0, 0.6);
+  padding: 7.6rem;
+  text-align: center;
+  margin-right: 21.5rem;
+}
+
+.slogan-ingredients-section {
+  font-family: "Yeseva One", sans-serif;
+  font-size: 3.6rem;
+  font-weight: 400;
+  color: #FFFFFF;
+  margin-bottom: 1.6rem;
+}
```

Comment:
Seems like we already have some similar set of styes. Looks like component. Let's extract this heading styles and have separate classes that could be combined. 
For example `class="heading-secondary heading-light"`

Reply:
Ok

Reply:
Such a heading will be encountered in future sections. But with this color in the last two sections there was no heading.

Reply:
assigned a new name to the selector

Reply:
Please notice that there're 2 classes: `heading-secondary heading-light`